frontends/web/src/components/guide/entry.tsx

```diff
@@ -54,7 +54,7 @@ export const Entry: FunctionComponent<TProps> = props => {
       <div className={[style.entryContent, shown ? style.expanded : ''].join(' ')}>
         {shown ? (
           <div className="flex-1">
-            {entry.text.trim().split('\n').map(p => <p key={p}>{p}</p>)}
+            {entry.text.trim().split('\n').map((p, idx) => <p key={idx}>{p}</p>)}
```

Comment:
> Multiple empty lines in an entry result in a duplicate key

this would only if the lines have same content correct?

Reply:
yeah see commit msg 

> Multiple empty lines in an entry result in a duplicate key.

Reply:
thanks for highlighting, I somehow didn't see the word "empty"